### ISSUE TITLE
MathJax "math_object_xx" magic has stopped working

### ISSUE THREAD

**Issue by rocky** [Mathics3/mathics-django#84]
For input `Red`

The following SVG is created:

```
<svg width="16.666666666666664px" height="16.666666666666664px" 
         xmlns:svg="http://www.w3.org/2000/svg"
          xmlns="http://www.w3.org/2000/svg" 
          version="1.1"
          viewBox="-0.333333 -0.333333 16.666667 16.666667">
     <rect x="0.000000" y="0.000000" width="16.000000" height="16.000000" 
               style="stroke: rgb(0.000000%, 0.000000%, 0.000000%); 
               stroke-opacity: 1.0; stroke-width: 0.666667px; 
               fill: rgb(100.000000%, 0.000000%, 0.000000%); fill-opacity: 1.0; opacity: 1.0" />
</svg>
```

And then this gets base64 encoded into MathML as: 

```
<mglyph width="16px" height="16px" src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYuNjY2NjY2NjY2NjY2NjY0cHgiIGhlaWdodD0iMTYuNjY2NjY2NjY2NjY2NjY0cHgiIHhtbG5zOnN2Zz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgICAgICAgICAgICAgICB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgICAgICAgICAgICAgICB2ZXJzaW9uPSIxLjEiCiAgICAgICAgICAgICAgICB2aWV3Qm94PSItMC4zMzMzMzMgLTAuMzMzMzMzIDE2LjY2NjY2NyAxNi42NjY2NjciPgogICAgICAgICAgICAgICAgPHJlY3QgeD0iMC4wMDAwMDAiIHk9IjAuMDAwMDAwIiB3aWR0aD0iMTYuMDAwMDAwIiBoZWlnaHQ9IjE2LjAwMDAwMCIgc3R5bGU9InN0cm9rZTogcmdiKDAuMDAwMDAwJSwgMC4wMDAwMDAlLCAwLjAwMDAwMCUpOyBzdHJva2Utb3BhY2l0eTogMS4wOyBzdHJva2Utd2lkdGg6IDAuNjY2NjY3cHg7IGZpbGw6IHJnYigxMDAuMDAwMDAwJSwgMC4wMDAwMDAlLCAwLjAwMDAwMCUpOyBmaWxsLW9wYWNpdHk6IDEuMDsgb3BhY2l0eTogMS4wIiAvPgo8L3N2Zz4K"/>
```

There is some javascript magic that was pulling the image out and turning it into: 

```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
   <mstyle displaystyle="true">
        <mstyle mathvariant="sans-serif">
            <mspace width="16px" height="16px" id="math_object_29"></mspace>
        </mstyle>
   </mstyle>
</math>
```

Somehow the mechanism to go from "math_object_29" to something that gets displayed is currently broken. 

If there is another way to get the SVG printed inside MathML mode, that would be fine too. In the past we were very back level in terms of MathJax so there might be a less hacky way to get this done.

@TiagoCavalcanteTrindade is this something you'd be willing to look at?

**Comment by TiagoCavalcanteTrindade** [comment]
@rocky I'll give it a look after I finish solving the zoom problem.

**Comment by rocky** [comment]
Many thanks.

**Comment by TiagoCavalcanteTrindade** [comment]
@rocky this is working to me.

In the middle of a lot of HTML there's an `img`:
```html
<img src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYuNjY2NjY2NjY2NjY2NjY0cHgiIGhlaWdodD0iMTYuNjY2NjY2NjY2NjY2NjY0cHgiIHhtbG5zOnN2Zz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgICAgICAgICAgICAgICB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgICAgICAgICAgICAgICB2ZXJzaW9uPSIxLjEiCiAgICAgICAgICAgICAgICB2aWV3Qm94PSItMC4zMzMzMzMgLTAuMzMzMzMzIDE2LjY2NjY2NyAxNi42NjY2NjciPgogICAgICAgICAgICAgICAgPHJlY3QgeD0iMC4wMDAwMDAiIHk9IjAuMDAwMDAwIiB3aWR0aD0iMTYuMDAwMDAwIiBoZWlnaHQ9IjE2LjAwMDAwMCIgc3R5bGU9InN0cm9rZTogcmdiKDAuMDAwMDAwJSwgMC4wMDAwMDAlLCAwLjAwMDAwMCUpOyBzdHJva2Utb3BhY2l0eTogMS4wOyBzdHJva2Utd2lkdGg6IDAuNjY2NjY3cHg7IGZpbGw6IHJnYigxMDAuMDAwMDAwJSwgMC4wMDAwMDAlLCAwLjAwMDAwMCUpOyBmaWxsLW9wYWNpdHk6IDEuMDsgb3BhY2l0eTogMS4wIiAvPgo8L3N2Zz4K" width="16px" height="16px">
```

![image](https://user-images.githubusercontent.com/62714153/122587975-e5f75300-d034-11eb-8081-1e3b8837557f.png)

**Comment by rocky** [comment]
@TiagoCavalcanteTrindade Investigating this further, I see that this _does_ work on google chorme, but not on firefox. 

In Firefox I see the message: 
```
Error in parsing MathML attribute value ‘3’ as length.  Attribute ignored.
```

every time this kind of MathML object gets rendered on Firefox (but not Chrome) and that my be why there is the problem. I tried Opera and also no problem.

**Comment by TiagoCavalcanteTrindade** [comment]
Ok, @rocky I'll install Firefox and check this out. Then I can also check if native MathML works properly in Firefox.

**Comment by rocky** [comment]
Fixed by #85
